Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -2708,6 +2739,17 @@
 			name = "Copy Plug-ins";
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		7BEDA69D28B396AC00B41C7C /* Product Dependencies */ = {
+			isa = PBXCopyFilesBuildPhase;
```

Comment:
Had to add this in order for the libWebKitPlatform.a to be depending to WebCore. Is this correct?

Reply:
Yeah, this looks correct. We need this since libWebKitPlatform.a doesn't link against WebCore, but it still needs to build after WebCore completes. (In an ideal world we'd have a way to indicate that it only depends on WebCore's API and not its binary, but this is not a regression from how WebCore would build.)